tests/system/test_system.py

```diff
@@ -542,13 +544,13 @@ def query_docs(client):
 @pytest.fixture
 def query(query_docs):
     collection, stored, allowed_vals = query_docs
-    query = collection.where("a", "==", 1)
+    query = collection.where(filter=FieldFilter("a", "==", 1))
```

Comment:
To the reviewers, this is an example of how the change in the `where` function.
It adds the keyword only arg `filter`, and you can see how it works before vs now.

Reply:
The old behavior without the filter arg is still expected to work, correct? Are there any system tests that still validate the old behavior or is that unit test only now?

Reply:
I added additional system tests here: https://github.com/googleapis/python-firestore/pull/698/commits/eff93287c649a0798a94f86e35c78534314722e2